configure.ac

```diff
@@ -2282,7 +2282,7 @@ AS_CASE(["$rb_cv_fiber_coroutine"], [yes|''], [
             rb_cv_fiber_coroutine=ppc64le
         ],
         [*], [
-            rb_cv_fiber_coroutine=
+            rb_cv_fiber_coroutine=ucontext
```

Comment:
@mame @nobu `ucontext` can work on most platforms, so we should add some test matrix for it. Is it possible to have `--fiber-coroutine=ucontext` passed to configure script? We already have `--disable-fiber-coroutine`. But soon that won't be an option, it will only be possible to have native implementation which falls back to `ucontext` by default. I don't know much about configure script or how to add.

Reply:
You cannot rely on `ucontext` being available, as not all platforms implement it (OpenBSD does not).  Ruby needs to continue to support the historical way of implementing fibers on platforms that do not implement `ucontext`, such as OpenBSD.

Reply:
On BSD it uses native implementations, what other situation is there?

Reply:
By native implementation, do you mean: something like `rb_cv_fiber_coroutine=amd64`?  If so, that isn't currently used on OpenBSD (I could see if it works on OpenBSD-amd64).  However, OpenBSD supports many more platforms than the platforms we have fiber coroutine support for.  We should keep supporting the historical fiber implementation on platforms that do not have native implementations and do not support `ucontext`.

Reply:
@jeremyevans What implementation does OpenBSD use? I'm trying to figure out right now. Native implementation should be used in every practical case.

Reply:
It looks like maybe it uses `setjmp` and `longjmp`?